Assets/HoloToolkit-Examples/Input/Scripts/InputTest.cs

```diff
@@ -32,6 +33,14 @@ public void OnInputClicked(InputClickedEventData eventData)
             eventData.Use(); // Mark the event as used, so it doesn't fall through to other handlers.
         }
 
+        /// <summary>
+        /// Just a public method called by the Unity OnClick Event.
+        /// </summary>
+        public void OnPointerClick(PointerEventData pointerEventData)
+        {
+            Debug.Log("OnPointerClick: " + pointerEventData.pointerId);
```

Comment:
Can we not use just "Debug.Log" everywhere.  This needs wrapping in it's own internal function so it's easier to control turning it off.  Possibly a separate PR.